plugins/MauticCitrixBundle/EventListener/CampaignSubscriber.php

```diff
@@ -256,7 +258,7 @@ public function onCampaignBuild(CampaignBuilderEvent $event)
                 'citrix.event.'.$product,
                 [
                     'label'           => 'plugin.citrix.campaign.event.'.$product.'.label',
-                    'formType'        => 'citrix_campaign_event',
+                    'formType'        => CitrixCampaignEventType::class,
```

Comment:
You have to keep 'formAlias' as described in 
https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit#

```
All ConfigSubscribers must keep 'formAlias' but also add 'formType' with the form type class name.
Before:
public function onConfigGenerate(ConfigBuilderEvent $event)
{
    $event->addForm([
        'bundle'     => 'EmailBundle',
        'formAlias'  => 'emailconfig',
        'formTheme'  => 'MauticEmailBundle:FormTheme\Config',
        'parameters' => $event->getParametersFromConfig('MauticEmailBundle'),
    ]);
}

After:
public function onConfigGenerate(ConfigBuilderEvent $event)
{
    $event->addForm([
        'bundle'     => 'EmailBundle',
        'formType'   => ConfigType::class,
        'formAlias'  => 'emailconfig',
        'formTheme'  => 'MauticEmailBundle:FormTheme\Config',
        'parameters' => $event->getParametersFromConfig('MauticEmailBundle'),
    ]);
}
```

Please check all Subscribers ;-)

Reply:
I'd gladly do that, but the document says it's only true for ConfigSubscribers.
This is not a ConfigSubscriber. Citrix bundle has no config subscribers.